Dockerfile

```diff
@@ -50,3 +50,12 @@ RUN chmod +x /startapp.sh
 COPY rc.xml.template /opt/base/etc/openbox/rc.xml.template
 
 WORKDIR /config
+RUN mkdir -p ./home/input ./home/output ./home/omero
```

Comment:
This needs to stay in Galaxy ... it creates a input/output dir in the CWD of Galaxy  and every job gets its own directory.

I don't think it will work if we do it in the container, since you don't know where the CWD (which is mounted into the container) will be